pkg/metric/metric.go

```diff
@@ -38,12 +38,19 @@ var (
 )
 
 // Type represents the type of a metric e.g. a counter. See
-// https://prometheus.io/docs/concepts/metric_types/.
+// https://prometheus.io/docs/concepts/metric_types/ and,
+// https://github.com/OpenObservability/OpenMetrics/blob/main/specification/OpenMetrics.md#metric-types.
 type Type string
 
 // Gauge defines a Prometheus gauge.
 var Gauge Type = "gauge"
 
+// Info defines an OpenMetrics info.
```

Comment:
Small nit on the comments, do you want to replace "Prometheus" in the counter and gauge comment as well?